application/controllers/admin/UserManagement.php

```diff
@@ -169,17 +170,17 @@ public function applyedit()
             Yii::app()->end();
         }
 
-        $aUser = $this->_editUser($aUser);
-        if ($aUser === false) {
+        $oUser = $this->updateAdminUser($aUser);
+        if ($oUser === false) {
             return Yii::app()->getController()->renderPartial('/admin/usermanagement/partial/json', ["data" => [
                 'success' => false,
                 'error' => print_r($oUser->getErrors(), true),
```

Comment:
Here $oUser === false , it's NOT a object

Reply:
ok